src/components/PersonCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonCard: React.FC<Props> = ({ person, people }) => {
+  const { slug } = useParams();
+
+  const mother = people.find((p) => p.name === person.motherName);
+  const father = people.find((p) => p.name === person.fatherName);
```

Comment:
Don't do this on every render. Do it on load or in `useMemo`

Reply:
Thx 😊 Done ✅